src/util/Algorithm.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Julian Mundhahs (mundhahj@informatik.uni-freiburg.de)
+
+#pragma once
```

Comment:
Please use proper Header guards, CLION creates them automatically:).
I know that there is a lot of `#pragma once` in the code.

Reply:
(from legacy times.)